tools/devtool

```diff
@@ -72,7 +72,7 @@
 DEVCTR_IMAGE_NO_TAG="public.ecr.aws/firecracker/fcuvm"
 
 # Development container tag
-DEVCTR_IMAGE_TAG="v44"
+DEVCTR_IMAGE_TAG="v48"
```

Comment:
Why this big of a jump?

Reply:
I simply used the devctr we currently use on main.

Reply:
A few version numbers went into specialized devctrs for the old 1.0 branch because we didn't upgrade that to 1.64, and then the 1.65 devctr never got backported to 1.1 and 1.2 branches.

Reply:
Yes, v44 was build for the upgrade to 1.64 in #3179 and v48 was build by me for upgrade to 1.66. Unfortunately no other intermediate upgrades have been backported so that's the reason for the big bump in devctr version.